content/docs/2.13/authentication-providers/aws-kiam.md

```diff
@@ -4,6 +4,8 @@ title = "Kiam Pod Identity for AWS"
 
 [**Kiam**](https://github.com/uswitch/kiam/) lets you bind an AWS IAM Role to a pod using an annotation on the pod.
 
+> ⚠️ **WARNING:** [AWS KIAM was abandoned](https://github.com/uswitch/kiam/#-%EF%B8%8Fthis-project-is-now-being-abandoned-%EF%B8%8F-) and support for it will be removed from KEDA on v2.15. We strongly encourage the migration to [`aws` auth](./aws.md).
```

Comment:
```suggestion
> ⚠️ **WARNING:** `aws-kiam` auth has been deprecated given [AWS KIAM is no longer maintained](https://github.com/uswitch/kiam/#-%EF%B8%8Fthis-project-is-now-being-abandoned-%EF%B8%8F-). As a result, [support for it will be removed from KEDA on v2.15](https://github.com/kedacore/keda/issues/5085). We strongly encourage the migration to [`aws` auth](./aws.md).
```

Reply:
See above, let's use discussion instead

Reply:
yeah

Reply:
done